### ISSUE TITLE
Helm Chart Grafana Oncall Celery Crash Problem with Rabbitmq User

### ISSUE THREAD

**Issue by melikesozeri** [grafana/oncall#782]
Hi everyone.
I'm trying to upgrade Grafana Oncall with Helm chart.(1.35.0 --> 1.49.0 ) I've set _rabbitmq.enabled=true_. But in celery pods crashs and  restarts every 10 seconds. How can I set rabbitmq user? In deployments no env variable for RABBITMQ_USERNAME, there is only RABBITMQ_PASSWORD. But in settings/base.py referred RABBITMQ_USERNAME env value for user.
I think, celery did not use default Rabbitmq user.  How can I define users and passwd in Helm chart? How can I solve that?
celery logs:
``` + celery --quiet -A engine worker -l info --concurrency=1 --max-tasks-per-child=100 -Q default,critical,long,slack,telegram,webhook,celery --beat
/usr/local/lib/python3.9/site-packages/celery/platforms.py:840: SecurityWarning: You're running the worker with superuser privileges: this is
absolutely not recommended!

Please specify a different user using the --uid option.
User information: uid=0 euid=0 gid=0 egid=0
  warnings.warn(SecurityWarning(ROOT_DISCOURAGED.format(
2022-11-04 13:53:46,655 source=engine:celery task_id=??? task_name=??? name=celery.worker level=CRITICAL Unrecoverable error: AccessRefused(403, 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.', (0, 0), '')
.....
.....
amqp.exceptions.AccessRefused: (0, 0): (403) ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.
```
rabbitmq logs:
`2022-11-04 13:49:42.374112+00:00 [error] <0.6076.0> PLAIN login refused: user 'None' - invalid credentials`
container configs:
```
kubectl exec -it release-oncall-celery-67c47c6cf7-cg9fh --container wait-for-db bash
./settings/base.py:RABBITMQ_USERNAME = os.getenv("RABBITMQ_USERNAME")
./settings/base.py:RABBITMQ_PASSWORD = os.getenv("RABBITMQ_PASSWORD")
```

**Comment by dinhdobathi1992** [comment]
hi, please try to remove the value of `passwordKey` in the values file. I'm making the issue for it currently

**Comment by iskhakov** [comment]
Hi, looks like this was fixed in 1.50.0
https://github.com/grafana/oncall/pull/775/files

Try the latest version please

**Comment by iskhakov** [comment]
Closing for now, please re-open if the problem persists in the newer versions
